migrations/20180427203643_export_job_field_size.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class ExportJobFieldSize extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+     *
+     * The following commands can be used in this method and Phinx will
+     * automatically reverse them when rolling back:
+     *
+     *    createTable
+     *    renameTable
+     *    addColumn
+     *    renameColumn
+     *    addIndex
+     *    addForeignKey
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function up()
+    {
+		/**
+		 * Changing fields and filters to the MEDIUM_TEXT
+		 * length. We would not (always) be able to handle
+		 * very large deployments with many fields
+		 * with a 255 varchar.
+		 */
+		$this->table('export_job')
+			->changeColumn(
+				'fields',
+				'text',
+				['null' => true, 'limit' => 16777215, 'default' => null]
+			)
+			->changeColumn(
+				'filters',
+				'text',
+				['null' => true, 'limit' => 16777215, 'default' => null]
+			)
+			->update();
+    }
+    public function down()
```

Comment:
Its failing because something is wrong with your migration. Have you tested downgrading to verify it works? 
Does it work with the test data in the DB?

Reply:
>   SQLSTATE[01000]: Warning: 1265 Data truncated for column 'fields' at row 1